### PR TITLE
Add release action workflow

### PR DESCRIPTION
**Pull request by vadzim-kazak**
The basic idea here is to trigger this workflow on tag push and perform the following steps:
- Create a new release. _ncipollo/release-action@v1_ uses tag description as a release description
- Build docker image and push it into public docker hub repo

### CODE REVIEW

File: .github/workflows/release-action.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: release-action
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
```
Comment (StefanoGuerrini) [inline .github/workflows/release-action.yaml]: Can we use a fixed version?

**Review by StefanoGuerrini — APPROVED** [review]
nice :)

File: .github/workflows/release-action.yaml (hunk 1 above)
Comment (vadzim-kazak) [reply .github/workflows/release-action.yaml]: yep, makes sense, thanks

**Review by frankkoornstra — COMMENTED** [review]
Don't know if you're the right person to ask but what's the reason we didn't go with Gitlab here?

**Comment by vadzim-kazak** [comment]
@frankkoornstra Frank, in my understanding we cannot do this because this is a public repo. I guess it won't be right to link it to our private infra.

**Comment by frankkoornstra** [comment]
Oh right, forgot about it being public :sweat_smile: Thanks for the reminder

**Review by frankkoornstra — APPROVED** [review]

**Comment by rdohms** [comment]
Its not that its "wrong" or "bad" to link to our private stuff.
But it is more "OSS friendly" and it allows users to see the same data we see.

**Review by rdohms — APPROVED** [review]
Not too relevant, but using Eve with public repos is just fine, so you could have had some of the release stuff from Eve if you wanted too.

**Pull request merged**